assets/diagram-js-minimap.css

```
@@ -31,14 +31,13 @@
 }
 
 .djs-minimap:not(.open) .toggle {
-  width: 46px;
-  height: 46px;
-  line-height: 46px;
+  padding: 10px;
   text-align: center;
 }
 
-.djs-minimap:not(.open) .toggle:before {
-  content: 'Open';
+.djs-minimap:not(.open) .toggle:before,
+.djs-minimap.open .toggle:before {

```

How about if we simplify it a bit?
```suggestion
.djs-minimap .toggle:before {
```

---

Actually, I did it exactly that way in the project I am working currently working on.
I just did not want to break other css rules or plugins by changing the specificity.
Maybe you should not simplify it, cause it may be a breaking change for some people...

---

I understand your concern, but cannot really think of an example where lowering specificity could be a breaking change. I imagine that if somebody overrides provided styles, custom CSS should have higher or at least equal specificity. Then, if we lower our styles' specificity, custom CSS should remain to work.

---

You are right.